datafusion-expr/src/lib.rs

```diff
@@ -16,12 +16,14 @@
 // under the License.
 
 mod aggregate_function;
```

Comment:
I also like how this reorganization is making the structure of `Expr` easier to understand

Reply:
these:
- #1774 
- #1784 

is also trying to tear up the cumbersome/bloated expr by traits.

Reply:
so that each trait module can be migrated separately

Reply:
> is also trying to tear up the cumbersome/bloated expr by traits.

🏅